htlcswitch/link.go

```diff
@@ -441,7 +441,8 @@ func (m *hookMap) invoke() {
 
 // hodlHtlc contains htlc data that is required for resolution.
 type hodlHtlc struct {
-	pd         *lnwallet.PaymentDescriptor
```

Comment:
I prolly would arrange this commit differently to reduce the size - for instance a single commit that removes this `pd` from `hodlHtlc` and its relevant changes.

Reply:
I will do this but I don't want to hear any complaints about diff churn as a result.